src/domains/sign/api_product/sign/_base_policy.xml

```diff
@@ -1,11 +1,6 @@
 <policies>
     <inbound>
         <base />
-        <choose>
-            <when condition="@(context.User.Groups.All(g =&gt; g.Name != &quot;ApiNoRateLimit&quot;))">
-                <rate-limit calls="150" renewal-period="5" remaining-calls-header-name="x-rate-limit-remaining" retry-after-header-name="x-rate-limit-retry-after" />
-            </when>
-        </choose>
```

Comment:
to protect our backend can we create a rate limit based on caller ip address?

Reply:
Yes, it's something we've planned to do. Since a caller could potentially have multiple IP addresses, would it be more effective to use the API Key as rate limit key?

Reply:
yep. the default is per subscription which is ok.

Reply:
it's the same without choose and when condition... but LGTM :)

```
<rate-limit calls="150" renewal-period="5" remaining-calls-header-name="x-rate-limit-remaining" retry-after-header-name="x-rate-limit-retry-after" />
```